keyboards/kradoindustries/kousa/info.json

```diff
@@ -0,0 +1,156 @@
+{
+    "keyboard_name": "Kousa-TKL", 
+    "manufacturer": "Krado Industries",
+    "url": "https://www.kradoindustries.com/", 
+    "maintainer": "Krado Industries", 
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x12D0",
+        "pid": "0x4B4F",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "GP10", "pin_b": "GP9" }
+        ]
+    },
+    "rgblight": {
+        "led_count": 18,
+        "pin": "GP4",
+        "hue_steps": 10,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
```

Comment:
These are defaults, and a value so close to default that it really doesnt offer much value
```suggestion
```